spec/factories/container_group.rb

```diff
@@ -1,5 +1,7 @@
 FactoryGirl.define do
   factory :container_group do
     sequence(:name) { |n| "container_group_#{seq_padded_for_sorting(n)}" }
+    association :ext_management_system, :factory => :ems_kubernetes
+    association :container_project, :factory => :container_project
```

Comment:
I am wary of this (as everyone should be wary of FactoryGirl associations...) because it doesn't appear that ContainerGroups require these associations. More associations by default == more processing and resources on any specs using this factory that _don't_ depend on these associations. Perhaps you should make them in the spec that they are needed for instead of doing them here?